torch/_meta_registrations.py

```diff
@@ -1799,8 +1799,7 @@ def meta_select(self, dim, index):
 
     check(
         not (-index > size or index >= size),
-        lambda: f"select(): index {index} out of range for tensor of size "
-        f"{self.size()} at dimension {dim}",
+        lambda: f"{self.size()} at dimension {dim}",
```

Comment:
can you restore the error message? (any reason you took away some info?)

Reply:
Added back, thanks for the catch!